client/ayon_core/hosts/max/plugins/publish/validate_instance_in_context.py

```diff
@@ -38,15 +38,15 @@ def process(self, instance):
             context_label = "{} > {}".format(*context)
             instance_label = "{} > {}".format(folderPath, task)
             message = (
-                "Instance '{}' publishes to different context than current "
-                "context: {}. Current context: {}".format(
+                "Instance '{}' publishes to different context(folder or task) "
```

Comment:
I'd personally just write it like this. Much easier to read, and artists understands it the same.
```suggestion
                "Instance '{}' publishes to different folder or task "
```